gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,654 @@
+
+To prevent potential timeout and connectivity errors when running procedures in this guide, your cluster must meet the <<k8-cluster-reqs>>. 
+
+. Run `kubectl config get-contexts` to list the clusters, for example:
++
+[source,kubectl]
+----
+kubectl config get-contexts
+----
++
+This command is particularly helpful if you have multiple Kubernetes clusters. You can use the command to get the name of the the correct cluster. For example, assume that you have a variety of clusters (`my-cluster`, `my-aks`, `my-gke`, `my-eks`). The command prints a list of your clusters. 
++
+The following _example_ shows a list of clusters produced by the command:
++
+----
+CURRENT   NAME          CLUSTER       AUTHINFO    NAMESPACE
+          my-cluster    my-cluster                my-cluster1
+          my-aks        my-aks                    my-aks
+          my-gke        my-gke                    my-gke
+          my-eks        my-eks                    my-eks
+----
++
+If your cluster is not listed and you are using a custom cluster context file on your machine to connect to your cluster, you might need to add its path to a `KUBECONFIG` environment variable that points to that file. By default, `kubectl` searches for a file named `config` in the `$HOME/.kube` directory. That file lists each `context`, which provides properties such as the cluster name, user, and certificate authority data. 
++
+You can check by running `echo $KUBECONFIG` from your terminal window, which prints a environment variable's value, similar to this example: 
++
+----
+{KUBECONFIG}:/Users/me/.kube/config:/Users/me/.kube/additional-clusters/my-gke:/Users/me/.kube/additional-clusters/my-aks:/Users/me/.kube/additional-clusters/my-eks
+----
++
+Adding the path to any additional cluster context files on your machine enables the `kubectl config get-contexts` to list them and for you to use them.
++
+For more detail, see the https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/[Kubernetes documentation^] on the `kubeconfig` file. 
+
+. Provide the name of a cluster into which you plan to deploy your gateway.
++
+The following example uses the cluster `my-eks`, identified in the cluster list example: 
++
+----
+kubectl config use-context my-eks
+----
++
+The command switches to the selected cluster context and prints a message similar to this one:
++
+----
+Switched to context "my-eks".
+----
++
+If you receive an error that begins `error: no context exists with the name`, ensure you are using the `NAME` of the cluster listed by `kubectl config get-contexts` and not the `CLUSTER` value.
+
+. Check that your cluster has one or more nodes:
++
+[source,kubectl]
+----
+kubectl get nodes
+----
++
+The command returns a output similar to this example:
++
+----
+NAME                                           STATUS   ROLES    AGE   VERSION
+ip-192-168-70-170.us-west-2.compute.internal   Ready    <none>   17h   v1.23.16-eks-48e63af
```

Comment:
```suggestion
ip-192-168-70-170.us-west-2.compute.internal   Ready    <none>   17h   v1.x.x-eks-48e63af
```

Reply:
IMHO, I would remove the `-eks-48e63af` part of the versions too